app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -124,23 +143,31 @@ def create_document_data(evss_claim_id, upload_data)
         )
       end
 
-      def upload_to_vbms(auth_headers, evss_claim_id, pdf_path, form_id)
+      def upload_to_vbms(pdf_path, form_id)
         upload_data = get_evss_claim_metadata(pdf_path, form_id)
         document_data = create_document_data(evss_claim_id, upload_data)
 
         raise Common::Exceptions::ValidationErrors, document_data unless document_data.valid?
 
-        if Flipper.enabled?(:disability_compensation_lighthouse_document_service_provider)
-          # TODO: create client from lighthouse document service
-        else
-          client = EVSS::DocumentsService.new(auth_headers)
-        end
+        # thin wrapper to isolate upload for logging
         file_body = File.open(pdf_path).read
-        client.upload(file_body, document_data)
+        perform_client_upload(file_body, document_data)
       ensure
         # Delete the temporary PDF file
         File.delete(pdf_path) if pdf_path.present?
       end
+
+      def perform_client_upload(file_body, document_data)
+        client.upload(file_body, document_data)
+      end
+
+      def client
```

Comment:
this minor refactor is more readable, and also makes client available in our `perform_client_upload` method without needing to pass it as an argument